packages/subgraph/README.md

```diff
@@ -20,51 +20,10 @@ Get started using the Subgraphs with an introduction and query examples:
 
 https://docs.superfluid.finance/superfluid/docs/subgraph
 
-# 👨‍🚀 Hosted Subgraphs
```

Comment:
What was the rationale behind the removal? I am for it, but it would be good to have a written notes from your thoughts @d10r .

Reply:
A few reasons:
* the info here (_All subgraphs are available via The Graph's hosted service:_) was stale and has become wrong since last updated, because not all subgraphs are available via the hosted service anymore
* there's no / very little explanation of the concepts used (v1 endpoits, dev endpoints, feature endpoints)
* the dev endpoints are at least partially unmaintained / lagging behind v1 endpoints

I could have replaced by something else, but don't currently have enough information to do so and didn't want to get derailed from the actual task of deprecating networks. We can anytime do in a dedicated PR.

I also briefly checked with @0xdavinchee if he's ok with this.